test/conformance/conformancetest_helper.go

```diff
@@ -70,12 +72,12 @@ func fetchRuntimeInfo(t *testing.T, clients *test.Clients, options *test.Options
 }
 
 // fetchEnvInfo creates the service using test_images/environment and fetches environment info defined inside the container dictated by urlPath.
-func fetchEnvInfo(t *testing.T, clients *test.Clients, urlPath string, options *test.Options) ([]byte, *test.ResourceNames, error) {
+func fetchEnvInfo(t *testing.T, clients *test.Clients, urlPath string, opts ...ServiceOption) ([]byte, *test.ResourceNames, error) {
```

Comment:
So fetchRuntimeInfo returns a map of all of the EnvVars set in the image as well. I am fine doing it later, but it would be nice to move to using it instead of fetchEnvInfo. It would allow us to delete a bunch of code and a test image.

Reply:
Let's follow up?

Reply:
Created new issue https://github.com/knative/serving/issues/3985